_data/reviews.yml

```
@@ -39,6 +39,13 @@
       feedback: Sonali gave me really good advice about how to approach being in a new backend team that wasn't turning out exactly as I had had expected, and how to go about job hunting when I am in this transitional state of moving from front end to backend. 
       name: Anonymous
       type: Ad-Hoc
+    - score: 5
+      feedback: |
+        I came to the session with lot of questions and indirection. But, Sonali was able to give me right suggestions and resources to navigate and move forward.
+      date-f: 5 July 2025
+      date: 20250705
+      name: Sowmiya Ravikumar
+      type: Ad-Hoc
```

To add the rating-only feedback:

```suggestion
      type: Ad-Hoc
    - score: 5
      date: 20250714
      date-f: 14 July 2025
      name: Kaushiki Agrawal
      type: Ad-Hoc
```